docs/snippets/package/azure/crossplane.yaml

```diff
@@ -1,13 +1,27 @@
-apiVersion: meta.pkg.crossplane.io/v1alpha1
+apiVersion: meta.pkg.crossplane.io/v1
 kind: Configuration
 metadata:
   name: getting-started-with-azure
   annotations:
+    meta.crossplane.io/maintainer: Crossplane Maintainers <info@crossplane.io>
+    meta.crossplane.io/source: github.com/crossplane/crossplane
+    meta.crossplane.io/license: Apache-2.0
+    meta.crossplane.io/description: |
+      An introductory example to Crossplane and Composition for Azure.
+    meta.crossplane.io/readme: |
+      An introductory example to Crossplane and Composition for Azure.
+      This will enable provisioning of an Azure SQL database instance.
+
+      [Install & 
+      Configure](https://crossplane.io/docs/v1.9/getting-started/install-configure.html)
```

Comment:
@jastang if we link to the docs, we'll need to make sure to update these every time we cut a release. We do a find and replace on a few different places during that process, but in order to identify that these need to be replaced, I think we should use https://crossplane.io/docs/master/getting-started/install-configure.html on `master` so that when we cut the `v1.10` branch (for example) our find and replace `s/master/v1.10/g` will catch this. What do you think?

Reply:
@hasheddan good call -- makes sense to me. I'll update throughout 👍